frontends/election-manager/src/app.test.tsx

```diff
@@ -638,7 +638,8 @@ test('tabulating CVRs', async () => {
 
   // Check L&A Materials are unavailable after live CVRs have been loaded
   fireEvent.click(getByText('L&A'));
-  expect(queryByText(zeroReportTitle)).toBeNull();
+  getByText('L&A materials are not available', { exact: false });
+  expect(queryByText(ZERO_REPORT_BUTTON_TEXT)).toBeNull();
```

Comment:
For readability, this alternative is often preferred:

```ts
expect(queryByText(ZERO_REPORT_BUTTON_TEXT)).not.toBeInDocument();
```

[Reference](https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument)